config/v1/types_operatorhub.go

```diff
@@ -6,10 +6,17 @@ import (
 
 // OperatorHubSpec defines the desired state of OperatorHub
 type OperatorHubSpec struct {
+	// disableAllDefaultSources allows you to disable all the default hub
+	// sources. If this is set to true, the disabled setting in sources is
+	// ignored.
+	// +optional
+	DisableAllDefaultSources bool `json:"disableAllDefaultSources,omitempty"`
 	// sources is the list of default hub sources and their configuration.
-	// If the list is empty, it indicates that the default hub sources are
-	// enabled on the cluster. The list of default hub sources and their
-	// current state will always be reflected in the status block.
+	// If the list is empty, it implies that the default hub sources are
+	// enabled on the cluster. If disableAllDefaultSources is true,
+	// the disabled setting in sources is ignored. The list of default hub
```

Comment:
I would think that if DisableAllDefaultSources is true that we could then say if a HubSource is marked specifically as active in the spec, then it will override the DisableAllDefaultSources? WDYT?

Reply:
I am fine with that.